src/constants.js

```diff
@@ -17,14 +17,23 @@ export const TestDataLocations = {
   RANDOM: "random"
 };
 
+const labelColor = "rgb(254, 96, 3)"; // was 186
+const labelColorSemi = "rgba(254, 96, 3, 0.4)";
+const featureColor = "rgb(75, 155, 213)";
+
 export const colors = {
   feature: "rgb(70, 186, 168)",
   label: "rgb(186, 168, 70)"
 };
 
```

Comment:
Ah I see - we both had similar ideas but implemented differently. Let's add standardizing and consolidating colors to our list of clean up as we churn on styles.